package.json

```diff
@@ -4,32 +4,24 @@
   "description": "Sequelize adapter for Node-Casbin",
   "main": "lib/adapter.js",
   "typings": "lib/adapter.d.ts",
-  "scripts": {
-    "precommit": "lint-staged",
-    "prepublish": "yarn run lint && yarn run test && yarn build",
-    "build": "rimraf lib && tsc",
-    "lint": "tslint \"src/**/*.ts\"",
-    "fix": "tslint \"src/**/*.ts\" --fix",
-    "test": "jest"
-  },
+  "scripts": {},
   "devDependencies": {
     "@types/jest": "^23.3.1",
     "@types/lodash": "^4.14.113",
     "@types/node": "^10.5.3",
     "@types/sequelize": "^4.27.24",
+    "coveralls": "^3.0.2",
     "husky": "^0.14.3",
     "jest": "^23.4.2",
     "lint-staged": "^7.2.0",
-    "rimraf": "^2.6.2",
```

Comment:
Why is ``rimraf`` deleted?

Reply:
This is the lib for fs' rm operating, like `rm -rf {yourPath}`. This repo no need to.

Reply:
ok